controllers/home.js

```diff
@@ -10,6 +10,9 @@ function home(req, res) {
   if (req.query.breach) {
     const reqBreachName = req.query.breach.toLowerCase();
     featuredBreach = req.app.locals.breaches.filter(breach => breach.Name.toLowerCase() === reqBreachName)[0];
```

Comment:
Technically outside the scope of this PR, but looking at this code again, I'm curious if using the `.find()` API here may be slightly more appropriate than using `.filter()[0]`.

Where:

> The `find()` method returns the **value** of the **first element** in the array that satisfies the provided testing function. Otherwise `undefined` is returned.

vs

> The `filter()` method creates a new array with all elements that pass the test implemented by the provided function.

Reply:
oh nice! This is way better. Pretty sure there are a few other places where this would also make more sense.